app/views/admin/users/edit.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1><%= t(".title") %></h1>
+<%= simple_form_for user, url: admin_user_path(user) do |f| %>
```

Comment:
Can we pull the from from here and the new view into a `_form` partial?

Reply:
No, I did have it with a partial but once users was changed to a singular resource I couldn't get it to work (just like the controller actions).